### PR TITLE
3.6  version OHOS adaptation

### PR DESCRIPTION
**Pull request by xqg243795168**
3.6  version OHOS adaptation

### CODE REVIEW

File: cocos/scripting/lua-bindings/script/framework/device.lua [hunk 1]
```diff
@@ -58,6 +58,8 @@ elseif target == cc.PLATFORM_OS_WINRT then
     device.platform = "winrt"
 elseif target == cc.PLATFORM_OS_WP8 then
     device.platform = "wp8"
+elseif target == cc.PLATFORM_OS_OHOS then
+    device.platform = "openharmony"
```
Comment (minggo) [inline cocos/scripting/lua-bindings/script/framework/device.lua]: Should change to `HarmonyOS Next`?

File: tests/lua-tests/project/Classes/AppDelegate.cpp [hunk 2]
```diff
@@ -43,15 +43,19 @@ bool AppDelegate::applicationDidFinishLaunching()
...
 #endif
         register_test_binding(L);
     }
     lua_pop(L, 1);
 
-
-    pEngine->executeScriptFile("src/controller.lua");
+	#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+	    FileUtils::getInstance()->addSearchPath("");
+    	pEngine->executeScriptFile("controller.lua");
```
Comment (minggo) [inline tests/lua-tests/project/Classes/AppDelegate.cpp:54]: - why add an empty path
- wrong indent

File: cocos/scripting/lua-bindings/script/framework/device.lua (hunk 1 above)
Comment (wanghui187) [reply cocos/scripting/lua-bindings/script/framework/device.lua]: fixed

File: tests/lua-tests/project/Classes/AppDelegate.cpp (hunk 2 above)
Comment (wanghui187) [reply tests/lua-tests/project/Classes/AppDelegate.cpp:54]: fixed

**Pull request merged**